packages/itwin/measure-tools/package.json

```diff
@@ -46,21 +47,18 @@
   ],
   "peerDependencies": {
     "@itwin/appui-abstract": "rc",
-    "@itwin/appui-layout-react": "rc",
```

Comment:
Since this is used to import "Toolbar" it needs to be either a real or peerDependency and given the static nature of pieces of the package I think it'll need to be a peer.